src/planscape/forsys/forsys_request_params.py

```diff
@@ -157,6 +157,33 @@ def __init__(self, request: HttpRequest):
             DbRequestParams._URL_WRITE_TO_DB, True)
         self.scenario = get_scenario_by_id(
             self.user, self._URL_SCENARIO_ID, params)
+        
+
+# Parameters for deciding whether a stand is eligible for treatment.
+# TODO: make it possible to parse these parameters from URL and/or read them 
+# from the DB.
+class StandEligibilityParams:
+    BUILDINGS_KEY = "buildings"
```

Comment:
I'd add a note about these - these appear to be constants that correspond to the different options that StandEligibilityParams can hold; but the constants are to be used in the actual requests to forsys - these themselves are not the parameters.

Reply:
They're necessary in RasterConditionFetcher and RasterConditionTreatmentEligibilitySelector. 

They aren't used in the forsys input dataframe.

I could move them to ForsysGenerationInput - thinking it through now.

Reply:
fixed - I've moved them from StandEligibilityParams to ForsysGenerationInput.